src/Grpc.Net.Client/Balancer/CompleteContext.cs

```diff
@@ -0,0 +1,45 @@
+﻿#region Copyright notice and license
+
+// Copyright 2019 The gRPC Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#endregion
+
+using System;
+using System.Net;
+using Grpc.Core;
+
+namespace Grpc.Net.Client.Balancer
+{
+    /// <summary>
+    /// Context used to signal a call is complete.
+    /// </summary>
+#if HAVE_LOAD_BALANCING
+    public
+#else
+    internal
+#endif
+        sealed class CompleteContext
+    {
+        /// <summary>
+        /// Gets or sets the <see cref="DnsEndPoint"/> a call was made with.
+        /// </summary>
+        public DnsEndPoint? EndPoint { get; set; }
```

Comment:
More properties might get added so I want to leave them as settable properties rather than adding multiple constructors.

Renamed property to `Address`.